test/src/base-url/tsconfig.json

```diff
@@ -0,0 +1,43 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "lib": [
+      "dom",
+      "es5",
+      "es2015.collection",
+      "es2015.iterable"
+    ],
+    "target": "es5",
+    "noImplicitAny": false,
+    "removeComments": true,
+    "noUnusedLocals": true,
+    "preserveConstEnums": true,
+    "declaration": true,
+    "sourceMap": true,
+    "inlineSources": true, // Required for debugging NPM published package.
+    "outDir": "dist/",
+    "experimentalDecorators": true,
+    "baseUrl": ".",
+    "paths": {
```

Comment:
I didn't see any of the 4 paths definied here inside the test folder codebase, normal ?

Reply:
Sorry, what i mean is something like in this article : https://netbasal.com/sexier-imports-in-typescript-e3c645bdd3c6

I didn't see `@project/core` inside your codebase test

Reply:
Ah, my fault, sorry. I did not make it because I lost scope here.

What i needed was only to include files in compilation/discovering process and did not take care of the fact that such things can also be used by the tool. I will makwe some more additions.

thanks for pointing it out.

Reply:
Any news @davidmpaz ?